projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -556,8 +556,8 @@ public function get_profile_link( $service_name, $connection ) {
 	public function get_display_name( $service_name, $connection ) {
 		$cmeta = $this->get_connection_meta( $connection );
 
-		if ( 'mastodon' === $service_name && isset( $cmeta['external_name'] ) ) {
-			return $cmeta['external_name'];
+		if ( 'mastodon' === $service_name && isset( $cmeta['external_display'] ) ) {
+			return $cmeta['external_display'];
```

Comment:
We want to have `@jetpack_reach@mastodon.social` instead of just `jetpack_reach`